botocore/args.py

```diff
@@ -771,3 +772,17 @@ def _compute_user_agent_appid_config(self, config_kwargs):
                 f'maximum length of {USERAGENT_APPID_MAXLEN} characters.'
             )
         config_kwargs['user_agent_appid'] = user_agent_appid
+
+    def _compute_sigv4a_signing_region_set(self, config_kwargs):
+        sigv4a_signing_region_set = config_kwargs.get(
+            'sigv4a_signing_region_set'
+        )
+        if sigv4a_signing_region_set is None:
+            sigv4a_signing_region_set = self._config_store.get_config_variable(
+                'sigv4a_signing_region_set'
+            )
+        if not sigv4a_signing_region_set:
+            raise botocore.exceptions.InvalidConfigError(
+                error_msg="sigv4a_signing_region_set configuration must be non-empty"
+            )
```

Comment:
This doesn't seem like something we should be evaluating. The default is `None` which means we're going to fail here with the default in every case. We only fallback to `*` in the handler which I believe was a specific design decision made in your last PR.